frontend/src/metabase/static-viz/containers/StaticChart/types.ts

```diff
@@ -8,3 +10,8 @@ export interface StaticChartProps {
   options: any;
   colors?: ColorPalette;
 }
+
+export interface IsomorphicChartProps {
```

Comment:
@alxnddr Can we move this to the previous PR? Pie chart needs it as well

Reply:
Done